test/react-native/features/fixtures/app/react_native_navigation_js/index.js

```diff
@@ -53,25 +53,21 @@ export default class AppScreen extends Component {
     const jsConfig = defaultJsConfig()
     const scenario = new Scenarios[scenarioName](configuration, scenarioMetaData, jsConfig)
     console.log(`  with config: ${JSON.stringify(configuration)} (native) and ${jsConfig} (js)`)
-    NativeModules.BugsnagTestInterface.startBugsnag(configuration)
-      .then(() => {
-        Navigation.setRoot({
-          root: {
-            stack: {
-              children: [
-                {
-                  component: {
-                    name: 'Home'
-                  }
-                }
-              ]
+    Navigation.setRoot({
+      root: {
+        stack: {
+          children: [
+            {
+              component: {
+                name: 'Home'
+              }
             }
-          }
-        })
-        Bugsnag.start(jsConfig)
-        this.state.scenario = scenario
-        scenario.run()
-      })
+          ]
+        }
+      }
+    })
+    this.state.scenario = scenario
```

Comment:
This should probably use `setState`, otherwise the app won't re-render

Reply:
Yeah, I noticed that some test fixtures used setState and others not.  I fairly lazily left this as-is, largely because I'd already had a few attempts at getting the build to pass - but I'll note it to look at. All the iOS navigation tests are currently skipped and I guess this could be a factor (but if nothing else we could make it consistent).